hieroplatform/src/main/java/org/hiero/utils/Randomness.java

```diff
@@ -0,0 +1,34 @@
+package org.hiero.utils;
+
+import org.apache.commons.math3.random.MersenneTwister;
+
+public class Randomness {
```

Comment:
static is hard because a PSG has a lot of state. It seems that this type of implementation is typically called getInstance(), but I don't mind changing the name. 
I'll document everything clearly and add an interface. Currently the class is initiated by the first call to getInstance. It is not clear to me who should initialize it. Does it really makes sense to separate the initialization from the 'createInstance()' (or getInstance()) methods?